eslint/babel-eslint-parser/test/index.js

```diff
@@ -1,51 +1,85 @@
-import assert from "assert";
-import espree from "espree";
+import path from "path";
 import escope from "eslint-scope";
 import unpad from "dedent";
 import { parseForESLint } from "../src";
-import assertImplementsAST from "./helpers/assert-implements-ast";
 
-const babelOptions = {
+const BABEL_OPTIONS = {
   configFile: require.resolve(
     "@babel/eslint-shared-fixtures/config/babel.config.js",
   ),
 };
+const ALLOWED_PROPERTIES = [
+  "importKind",
```

Comment:
I'm assuming we want to leave these properties since they might be useful for Flow-related ESLint rules. `exportKind` is already used in [`eslint-plugin-flowtype`](https://github.com/gajus/eslint-plugin-flowtype/search?q=exportKind&unscoped_q=exportKind), so it seems like we should include both properties.

Reply:
`eslint-plugin-import` definitely uses these.

Reply:
We might want to add tests for third-party ESLint plugins in the e2e Circle CI workflow.